tests/integration/test_multiple_disks/test.py

```diff
@@ -193,6 +226,7 @@ def test_system_tables(start_cluster):
             "volume_type": "JBOD",
             "max_data_part_size": "1024000000",
             "move_factor": 0.1,
+            "disable_merges": 0,
         },
```

Comment:
worth adding a test for RAID1?

Reply:
I think it does not, it is experimental thing and can not be added from configuration.